### DISCUSSION TITLE
Color reversal on pages

### DISCUSSION THREAD

**Discussion by PhysicsPhun** [Pixel-Prowess/GitHub360-Summer24#11] [General]
How did you reverse your pages to have the dark background and white text?

**Comment by PhysicsPhun** [comment]
And to answer my own question:  https://github.com/settings/appearance
and uncheck synch and set to dark mode.  A lot better this way.

**Reply by planetoftheweb** [reply]
Yes, some developers love the dark mode. It's definitely less bring when all you do is look at a screen all day.
